.github/workflows/docker_builds.yml

```diff
@@ -0,0 +1,16 @@
+name: Publish Docker
+on: [push]
```

Comment:
I would rather spec push to master
https://github.com/PyTorchLightning/pytorch-lightning/blob/4d24032ea5d28dca4b3a7aefece2fd28572f7b10/.github/workflows/pypi-release.yml#L7-L12